webapp/src/auth.ts

```diff
@@ -398,19 +398,28 @@ export async function deleteAccount() {
 
     const userId = getState()?.profile?.id;
 
-    await apiAsync('/api/user', null, 'DELETE');
-
-    // Clear csrf token so we can no longer make authenticated requests.
-    pxt.storage.removeLocal(CSRF_TOKEN);
-
-    // Convert cloud-saved projects to local projects.
-    await cloud.convertCloudToLocal(userId);
-
-    // Update state and UI to reflect logged out state.
-    clearState();
-
-    // Reload page
-    window.location.reload();
```

Comment:
no more reload necessary?

Reply:
I removed it to see what would happen. All seems to work. Do you have any concerns?